package.json

```diff
@@ -39,6 +40,7 @@
   "devDependencies": {
     "@commitlint/cli": "17.0.3",
     "@commitlint/config-conventional": "17.0.3",
+    "@gnosis.pm/safe-contracts": "1.3.0",
```

Comment:
Why this package and not -> 

https://github.com/safe-global/safe-contracts

is there a difference?